webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -174,10 +175,7 @@ export class ConnectionOptionsTabService extends Bootstrap {
     const driver = await this.dbDriverResource.load(state.config.driverId, ['includeProviderProperties']);
 
     state.config.authModelId = driver?.defaultAuthModel;
-
-    state.config.configurationType = driver?.configurationTypes.includes(DriverConfigurationType.Manual)
-      ? DriverConfigurationType.Manual
-      : DriverConfigurationType.Url;
+    state.config.configurationType = driver ? getDefaultConfigurationType(driver) : DriverConfigurationType.Manual;
 
     state.config.host = driver?.defaultServer || 'localhost';
     state.config.port = driver?.defaultPort;
```

Comment:
defaultServer/Port/Database/Host; serverName is deprecated, we need to replace old approach with new one and send this properties via `mainProperties`